### PR TITLE
Golden outfit pull

### PR DESCRIPTION
**Pull request by pitis91**

### CODE REVIEW

**Comment by pitis91** [comment]
to make it more good if someone knows how to create action in the memorial to appear the player who has bought it it will be appreciated XD

**Comment by pitis91** [comment]
storages.lua

lines added:
666		-- Golden Outfit
667		GoldenBaseOutfit = 50580,
668		GoldenFirstAddon = 50581,
669		GoldenSecondAddon = 50582,

File: data/npc/scripts/Emperor Kruzak.lua [hunk 1]
```diff
@@ -7,6 +7,81 @@ function onCreatureDisappear(cid)		npcHandler:onCreatureDisappear(cid)			end
...
+		elseif npcHandler.topic[cid] == 6 then
+			if(doPlayerRemoveMoney(cid, 250000000) and player:getStorageValue(Storage.OutfitQuest.GoldenBaseOutfit) == 1 and player:getStorageValue(Storage.OutfitQuest.GoldenFirstAddon) < 1) then
+				npcHandler:say("Take this boots as a token of great gratitude. Let us forever remember this day, my friend. ", cid)
+				npcHandler.topic[cid] = 0
+				player:addOutfitAddon(1210, 1)
+				player:addOutfitAddon(1211, 1)
+				player:setStorageValue(Storage.OutfitQuest.GoldenFirstAddon, 1)
+			end
+		elseif npcHandler.topic[cid] == 7 then
+			if(doPlayerRemoveMoney(cid, 250000000) and player:getStorageValue(Storage.OutfitQuest.GoldenBaseOutfit) == 1  and player:getStorageValue(Storage.OutfitQuest.GoldenSecondAddon) < 1) then
```
Comment (ghost) [inline data/npc/scripts/Emperor Kruzak.lua]: player:removeMoney(250000000)

same with others

File: data/npc/scripts/King Tibianus.lua [hunk 2]
```diff
@@ -98,6 +173,103 @@ npcHandler:setMessage(MESSAGE_GREET, 'I greet thee, my loyal subject |PLAYERNAME
 npcHandler:setMessage(MESSAGE_FAREWELL, 'Good bye, |PLAYERNAME|!')
 npcHandler:setMessage(MESSAGE_WALKAWAY, 'How rude!')
 
+local focusModule = FocusModule:new()
+focusModule:addGreetMessage('hail king')
```
Comment (ghost) [inline data/npc/scripts/King Tibianus.lua:191]: those lines are already in the end of the file

**Review by ghost — CHANGES_REQUESTED** [review]

**Comment by ghost** [comment]
also can you please add the golden outfit display to each part of the addon? so the npc will it it along with the addon/outfit, check items ID 36336 to 36351

File: data/npc/scripts/King Tibianus.lua (hunk 2 above)
Comment (pitis91) [reply data/npc/scripts/King Tibianus.lua:191]: what i should do, delete them? i just spent 6hours to make work that but i'm not rlly a scripter XD just "copy-pasted" the nobleman irmana function and tryed till make the script work xD

File: data/npc/scripts/Queen Eloise.lua [hunk 3]
```diff
@@ -7,47 +7,89 @@ function onCreatureDisappear(cid)		npcHandler:onCreatureDisappear(cid)			end
...
+end
+
+npcHandler:setCallback(CALLBACK_MESSAGE_DEFAULT, creatureSayCallback)
+npcHandler:addModule(FocusModule:new())
+
 -- Promotion
 local node1 = keywordHandler:addKeyword({'promot'}, StdModule.say, {npcHandler = npcHandler, onlyFocus = true, text = 'I can promote you for 20000 gold coins. Do you want me to promote you?'})
 	node1:addChildKeyword({'yes'}, StdModule.promotePlayer, {npcHandler = npcHandler, cost = 20000, level = 20, text = 'Congratulations! You are now promoted.'})
 	node1:addChildKeyword({'no'}, StdModule.say, {npcHandler = npcHandler, onlyFocus = true, text = 'Alright then, come back when you are ready.', reset = true})
--- Postman
```
Comment (EPuncker) [inline data/npc/scripts/Queen Eloise.lua]: don't delete this part

File: data/npc/scripts/King Tibianus.lua [hunk 4]
```diff
@@ -102,3 +177,95 @@ local focusModule = FocusModule:new()
 focusModule:addGreetMessage('hail king')
 focusModule:addGreetMessage('salutations king')
 npcHandler:addModule(focusModule)
+
+			end
```
Comment (EPuncker) [inline data/npc/scripts/King Tibianus.lua]: move all this block to line 77

**Review by ghost — COMMENTED** [review]
test this one https://pastebin.com/raw/j0qvWirX and adapt it to other 2 npc

**Comment by miezaF** [comment]
Npcs doesnt work. Test it.

**Pull request closed without merging**